src/main/java/controllers/MemberFactoryUtil.java

```diff
@@ -0,0 +1,35 @@
+package controllers;
+
+import models.member.Member;
+
+public class MemberFactoryUtil {
```

Comment:
Can we name it to just MemberFactory?

Reply:
That was my original name, but the Code Quality check failed me, and asked me to add a Util at the end. Not sure how I can change it and why only this one has a problem.

Reply:
Wtf LOL Then keep it as it is now. Can you tell me which line in CheckStyle fails this?

Reply:
It's not Checkstyle, it's the other checker. The Code Quality one! How to fix? How did you even set it up?

Reply:
Wot HAHAS How does that even work okay then never mind! I will take a look at the code quality one later